components/cloud/types/customer.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+//import {LogoutAction} from './user';
+
+export type Customer = {
+    id: string;
+    name: string;
+    email: string;
+    billing_address: Address;
+    company_address: Address;
+    payment_method: PaymentMethod;
+}
+
+export type Address = {
+    city: string;
+    state: string;
+    country: string;
+    postal_code: string;
+    line1: string;
+    line2: string;
+}
+
+export type PaymentMethod = {
+    type: string;
+    last_four: number;
+    exp_month: number;
+    exp_year: number;
+    card_brand: string;
+}
+
+export const SET_CUSTOMER = 'SET_CUSTOMER';
+
+export type SetCustomerAction = {
+    type: typeof SET_CUSTOMER;
+    customer: Customer;
+};
+
+//export type CustomerActionType = SetCustomerAction | LogoutAction;
```

Comment:
I believe this overlaps with @devinbinnie [`Customer` type definition](https://github.com/devinbinnie/mattermost-redux/blob/727f469505d95b70060c93374adbfe878ab8e951/src/types/cloud.ts#L24) on this Redux PR https://github.com/mattermost/mattermost-redux/pull/1249 . We should use that instead and avoid duplicates in the webapp

Reply:
For future reference @nickmisasi, we should try to keep all our types in `mattermost-redux`, especially if they represent data coming from the server.

Reply:
FYI, that PR is merged so we can use that customer type here instead

Reply:
Should be good here. I moved the sku's into `types/cloud/sku.ts` but removed this file in favour of Devin's mattermost-redux changes